src/components/ui/ZoomControl/index.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import Button from 'components/ui/Button';
+import './style.scss';
+
+
+class ZoomControl extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      value: props.value,
+      className: props.className
+    };
+
+    // BINDINGS
+    this.triggerZoomIn = this.triggerZoomIn.bind(this);
+    this.triggerZoomOut = this.triggerZoomOut.bind(this);
+  }
+
+  triggerZoomIn() {
+    const newValue = this.state.value + 1;
+    if (newValue <= this.props.maxValue) {
+      this.setState({ value: newValue });
+      this.props.onZoomIn(this.state.value);
```

Comment:
Be careful, `this.setState` is asynchronous so `this.props.onZoomIn(this.state.value);` should be on the callback of `this.setState`. The same with the ZoomOut